packages/mcp-server/src/db.ts

```
@@ -49,17 +50,30 @@ function getDbPath(): string {
+function assertFts5Available(db: Database.Database): void {
+  try {
+    db.prepare('CREATE VIRTUAL TABLE _fts5_check USING fts5(x)').run();
+    db.prepare('DROP TABLE _fts5_check').run();
+  } catch (err) {
+    const message = err instanceof Error ? err.message : String(err);
+    throw new Error(
+      `FTS5 module is not available in this SQLite build.\n` +
+        `The Readied database uses FTS5 for full-text search triggers.\n` +
+        `Without FTS5, write operations (create/update/delete notes) will fail.\n` +
+        `Original error: ${message}`
+    );
```

ESLint

[error] There is no `cause` attached to the symptom error being thrown.

(preserve-caught-error)